nservicebus/upgrades/5to6-moving-away-from-ibus.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Moving away from `IBus` in NServiceBus Version 6
+summary: Describes how to send messages without the IBus
+---
+
+Starting with Version 6, the `IBus` interface has been deprecated.
+
+The `IBus` interface was one of the key interfaces when using previous versions of NServiceBus. It provided access to many of the operations like sending messages, subscribing to events, and manipulating headers. The `IBus` interface was available through dependency injection in message handlers, any custom components that were registered in the container and through other mechanisms like the saga base class.
+
+Some scenarios involving `IBus` include: endpoint creation, sending messages during endpoint startup, sending messages from within message handlers, using the injected `IBus` in custom components also registered in the container. 
+
+## During Endpoint Creation
+
+In previous versions of NServiceBus to start a new instance of an endpoint, either the `Bus` static class or the `IStartableBus` interface was used. In Version 6 these two concepts have been replaced. More information can be found in the [hosting](/nservicebus/hosting/) section of the documentation. Instead of the `Bus` class, a new class called `Endpoint` helps to start the endpoint.
```

Comment:
> In Version 6 these two concepts have been replaced. More information can be found in the [hosting](/nservicebus/hosting/) section of the documentation.

I think the "more information..." needs to come before the "these two concepts have been replaced". Since the more info is related to "start a new instance of an endpoint" not "two concepts have been replaced".